Modules/_sqlite/module.c

```diff
@@ -46,7 +46,8 @@ module _sqlite3
 PyDoc_STRVAR(module_connect_doc,
 "connect($module, /, database, timeout=5.0, detect_types=0,\n"
 "        isolation_level='', check_same_thread=True,\n"
-"        factory=ConnectionType, cached_statements=128, uri=False)\n"
+"        factory=ConnectionType, cached_statements=128, uri=False, *,\n"
+"        autocommit=sqlite3.LEGACY_TRANSACTION_CONTROL)\n"
```

Comment:
Any reason for making the new *autocommit* parameter keyword-only? (I am not against it, just curious.)

Reply:
Yes; keeping it available as a positional parameter makes no sense. In fact all parameters after *database* should've been keyword only parameters. There is an ongoing effort to make this happen, but it will require the required deprecation cycle of two releases before we can make any changes. For this case, maybe a longer deprecation cycle is better.

Serhiy brought up this issue [earlier this year](https://github.com/python/cpython/issues/93057), and I've started to pursue it, but it is a complicated matter involving Argument Clinic changes. Before I land them, we'd have to land gh-96178, so we can properly future test clinic changes, such as Serhiy's issue. An issue seldom comes alone :)